### DISCUSSION TITLE
[Bug]: Game - Pokemon FireRed. The emulator loads the same save ignoring its absence.

### DISCUSSION THREAD

**Discussion by Zkera** [visualboyadvance-m/visualboyadvance-m#1346] [General]
### Expected behavior?

after importing the .sav file and rebooting, open a new save

### Actual behavior?

opens the same save regardless of actions

### Steps to reproduce the problem

1. Replace the save file with a new one
2. Open the emulator
3. Open the GBA file
4. Done

### What game were you playing

<Some of this information can be provided with a screenshot of Rom Information, SAVE TYPE for GBA titles can be obtained from Options > Game Boy Advance > Config>
Title: Pokemon - Fire Red Version (USA, Europe)
GB, GBC or GBA: GBA
ROM CODE: 
BIOS Used?: no
SAVE TYPE: Attach Battery 
Clean

### What settings are you using?

Video driver (simple, opengl):
Sound driver (directsound, xaudio, openal, SDL):
Scale Filter Used?:

### Specifications and Build details

VBA-M: (include commit version number and whether you are using the 32-bit or 64-bit version)
OS: (include version and OS build number using WinVer if on Windows):
If using Linux, specify if using xorg or Wayland:
CPU:
GPU:
RAM:

### Copyright Compliance

- [X] There are no infringing files attached to this bug report.

**Comment by Squall-Leonhart** [comment]
I use the import feature quite often and it works fine, can you provide the save file, and fill out the build details

**Comment by Zkera** [comment]
I don't know why, but it fixed. I just tried to do the exact same thing, and the new save was imported
